tensorflow/c/kernels_test.cc

```diff
@@ -96,9 +102,11 @@ namespace tensorflow {
 
 static std::unique_ptr<OpKernel> GetFakeKernel(const char* device_name,
                                                const char* op_name,
+                                               const char* kernel_name,
```

Comment:
This feels like a misnomer, since the node's name isn't the necessarily the name of the kernel; it's typically a uniquified name based off of TF scopes/op name/custom user-provided names/etc, so let's name this "node name" instead.

Reply:
Thanks for the comments Brian! I also changed the naming for some of the variables elsewhere to clarify that it's node_name and not really the name of the kernel.